pytest_rerunfailures.py

```diff
@@ -186,13 +195,24 @@ def get_reruns_delay(item):
     rerun_marker = _get_marker(item)
 
     if rerun_marker is not None:
+        args = copy(rerun_marker.args).pop(0)
+
         if "reruns_delay" in rerun_marker.kwargs:
             delay = rerun_marker.kwargs["reruns_delay"]
-        elif len(rerun_marker.args) > 1:
-            # check for arguments
-            delay = rerun_marker.args[1]
+        elif len(args):
```

Comment:
`len(args)` is redundant, can just do `elif args` for non-empty I believe

Reply:
Dropped this bit, I'm using the same strategy as the existing code for this.